frontend/src/app/main/ui/dashboard/grid.cljs

```diff
@@ -109,10 +109,12 @@
            (dom/stop-propagation event)
            (st/emit! (modal/show
                       {:type :confirm
-                       :message (t locale "modals.add-shared-confirm.message" (:name file))
-                       :title (t locale "modals.add-shared-confirm.title")
+                       :message ""
+                       :title (t locale "modals.add-shared-confirm.message" (:name file))
                        :hint (t locale "modals.add-shared-confirm.hint")
+                       :cancel-label ""
```

Comment:
why not just omit the :cancel-label in this case? or use `nil` instead of `""`

Reply:
If you omit cancel-label, then the default "Cancel" label is used (see confirm.cljs). We can use other system, but there should be a way to distinguish between "the default label" or "no label and no button".

Reply:
See https://tree.taiga.io/project/uxboxproject/issue/760

Reply:
Hmm in case we want allow omiting cancel button, i think is better have `:default` value for default and omit it by default. Empty string is very confusing as option for ommit it.

Reply:
fixed!